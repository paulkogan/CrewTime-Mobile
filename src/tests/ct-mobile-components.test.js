import React, {} from "react";
//import {BrowserRouter, Route, Switch, Link, NavLink } from "react-router-dom";
import TimeEntryPage, {} from "../components/ct-timeentry-page"
import renderer from 'react-test-renderer';


// use: npm test -- -u`
it('renders Time EntryPage correctly', () => {
  const tree = renderer
    .create(<TimeEntryPage nlink={'kim99'}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});




const add = (a,b) => a+b;
test("should add two numbers", () => {

          const result = add(3,4);
          expect(result).toBe(7);


});
