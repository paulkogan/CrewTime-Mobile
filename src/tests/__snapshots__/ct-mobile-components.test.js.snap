// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Time EntryPage correctly 1`] = `
<div
  className="outer-div"
>
  <div
    className="indent-div"
  >
    2018-08-27
    <p
      className="prom-name"
    >
      undefined undefined
    </p>
    <div
      className="outer-div"
    >
      <div
        className="items-list"
      >
        <b>
          Add a New Work Time
        </b>
        <br />
        Building:  
        <div>
          <select
            name="selected_prop_id"
            onChange={[Function]}
            value={0}
          />
        </div>
        <br />
        <br />
        <br />
      </div>
      <div />
    </div>
  </div>
</div>
`;
